package.json

```diff
@@ -4,28 +4,25 @@
   "private": true,
   "license": "AGPL-3.0-or-later",
   "scripts": {
-    "build": "webpack --mode production",
-    "build-ci": "node --max_old_space_size=4096 -r ts-node/register/transpile-only scripts/ci-build.ts",
+    "build": "gulp build",
+    "build-ci": "gulp ci",
     "build-storybook": "build-storybook -s public --quiet",
-    "build:base": "webpack --mode production",
-    "build:chromium": "webpack --mode production --chromium",
-    "build:e2e": "webpack --mode production --e2e",
-    "build:firefox": "webpack --mode production --firefox --reproducible-build",
-    "build:gecko": "webpack --mode production --firefox-gecko",
-    "build:ios": "run-s build:ios:webpack build:ios:ext:prebuilt",
-    "build:ios:ext:prebuilt": "ts-node scripts/webextensiom-shim-prebuilt.ts",
-    "build:ios:webpack": "webpack --mode production --wk-webview",
+    "build:base": "gulp build",
+    "build:chromium": "gulp build --chrome",
+    "build:e2e": "gulp build --e2e",
+    "build:firefox": "gulp build --fx",
+    "build:gecko": "gulp build --android",
+    "build:ios": "gulp build --ios",
     "compile:contracts": "./scripts/contracts.sh",
```

Comment:
use `gulp` impl?

Reply:
@guanbinrui?

Reply:
the task not migrate?

Reply:
leave that in future maybe. it's not a frequent op and is isolated to other tasks